src/components/Footer/subscribeSection.js

```diff
@@ -39,89 +40,111 @@ const SubscribeSection = ({ size }) => {
-    message === 'success' ? (
-      <Box>
-        {messageSwitch(message)}
-      </Box>
-    ) : (
-      <Box >
-        <Grid container className={size === 'lg' ? classes.subscribeSectionLarge : classes.subscribeSectionSmall}  >
-          <Grid   className={size === 'lg' ? null : classes.newsLetter} >
-            <Typography
-              variant='body2'
-              color='textSecondary'
-              className={size !== 'lg'
+  return message === 'success' ? (
+    <Box>{messageSwitch(message)}</Box>
+  ) : (
+    <Box>
+      <Grid
+        container
+        className={
+          size === 'lg'
+            ? classes.subscribeSectionLarge
+            : classes.subscribeSectionSmall
+        }
+      >
+        <Grid className={size === 'lg' ? null : classes.newsLetter}>
+          <Typography
+            variant='body2'
+            color='textSecondary'
+            className={
+              size !== 'lg'
                 ? `${classes.sectionPaddingSm}`
                 : `${classes.sectionPaddingLg}`
-              }
-            >
-              Newsletter
-            </Typography>
-            <Typography
-              variant='body1'
-              color='textSecondary'
-              className={size !== 'lg'
+            }
+          >
+            Newsletter
+          </Typography>
+          <Typography
+            variant='body1'
+            color='textSecondary'
+            className={
+              size !== 'lg'
                 ? `${classes.sectionPaddingSm}`
                 : `${classes.sectionPaddingLg}`
-              }
-            >
-              To receive updates about new projects and trending topics on the index, subscribe here.
-            </Typography>
-          </Grid>
-          <form   onSubmit={submitEmail} >
-            <Typography
-              variant='body2'
-              color='textSecondary'
-              className={size !== 'lg'
+            }
+          >
+            To receive updates about new projects and trending topics on the
+            index, subscribe here.
+          </Typography>
+        </Grid>
+        <form onSubmit={submitEmail}>
+          <Typography
+            variant='body2'
+            color='textSecondary'
+            className={
+              size !== 'lg'
                 ? `${classes.sectionPaddingSm}`
                 : `${classes.sectionPaddingLg}`
-              }>
-                E-mail
-            </Typography>
-            <Grid className={size === 'lg' ? null : classes.emailSubscribe} >
-              <TextField
-                onInput={(e) => setInputValue(e.target.value)}
-                placeholder='name@domain.com'
-                type='email'
-                variant='outlined'
-                className={size !== 'lg'
+            }
+          >
+            E-mail
+          </Typography>
+          <Grid className={size === 'lg' ? null : classes.emailSubscribe}>
+            <TextField
+              id='searchTextfield'
```

Comment:
Is this id value necessary to pass the accessibility test?  If so, then perhaps we could rename it to be more specific (such as ```emailInput```)